Calligraphy/Controllers/QuoteController.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Calligraphy.Business.Quote;
+using Calligraphy.Data.Models;
+using Microsoft.AspNetCore.Cors;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using System.Net.Mime;
+
+namespace Calligraphy.Controllers
+{
+        [EnableCors("ApiCorsPolicy")]
+        [ApiController]
+        [Route("[controller]")]
+        public class QuoteController : ControllerBase
+        {
+        private readonly IQuoteService _quoteService;
+
+        public QuoteController(IQuoteService quoteService)
+        {
+            _quoteService = quoteService;
+        }
+
+        // GET ALL
+        [HttpGet]
+        [Route("/api/Quote")]
+        [Produces(MediaTypeNames.Application.Json)]
+        public IEnumerable<QuoteEntity> GetAll()
+        {
+            return _quoteService.GetAll();
+        }
+
+        // GET BY FORM ID
+        [HttpGet]
+        [Route("/api/Quote/{id:int}")]
+        [Produces(MediaTypeNames.Application.Json)]
+        public IActionResult GetByFormId(int id)
+        {
+            return _quoteService.GetByFormId(id);
+        }
+
+        // POST
+        [HttpPost]
+        [Route("/api/Quote")]
+        [Consumes(MediaTypeNames.Application.Json)]
+        public IActionResult Create([FromBody] QuoteEntity quote)
+        {
+            var result = _quoteService.Create(quote);
+            if (result)
+            {
+                return Ok(quote);
+            }
+            return BadRequest();
+        }
```

Comment:
If the generation of a quote is tied to the posting of a form, similar to how mailer has been implemented, can you explain why creating a quote needs a separate endpoint? The quote generation and initial values are influenced by the form is it not?

Reply:
I see your point, thanks!

Reply:
I fixed this issue and am now re-issuing this PR